src/service/packet_router.proto

```diff
@@ -52,10 +52,38 @@ message packet_router_register_v1 {
   bytes signature = 3;
 }
 
+// Session offer from the packet router to the gateway. If the gateway wants to
+// initiate a session key it should send a packet_router_session_init_v1 message
+// to the packet router using data in this offer to propose a session key.
+//
+// This message is sent by packet router after the register command is sent by
+// the gateway.
+message packet_router_session_offer_v1 { bytes data = 1; }
```

Comment:
Can this please be called "nonce" instead of "data". That more accurately describes what this field does, and since we're super sticklers about never renaming a protobuf field after its deployment, this is our last chance to change it.

Reply:
Fixed in [1e49483](https://github.com/helium/proto/pull/359/commits/1e4948358077027813cd81b70b30d20e67aa5901)

Reply:
ty!